ws-security-stax/src/main/java/org/apache/wss4j/stax/impl/processor/input/SecurityContextTokenInputHandler.java

```diff
@@ -108,9 +108,22 @@ public String getId() {
         wsInboundSecurityContext.registerSecurityTokenProvider(identifier, securityTokenProviderDirectReference);
 
         //fire a tokenSecurityEvent
+        SecurityContextTokenSecurityEvent securityEvent = createTokenSecurityEvent(securityContextTokenType, securityTokenProvider);
+        wsInboundSecurityContext.registerSecurityEvent(securityEvent);
+    }
+
+    private QName extractElementName(JAXBElement<?> jaxbElement) {
+        QName qName = jaxbElement.getName();
+        return new QName(qName.getNamespaceURI(), qName.getLocalPart());
+    }
```

Comment:
This method can be removed, and instead just do:

final QName elementName = securityContextTokenTypeJAXBElement.getName();